packages/react-sdk/src/css/normalize.ts

```diff
@@ -127,6 +127,14 @@ export const body = [
       value: `system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'`,
```

Comment:
We need a way to select this normalized fallback from ui.

Reply:
I don't understand what you mean

Reply:
got it, normalized the font